src/compass/utils/lut.py

```diff
@@ -362,3 +367,55 @@ def resample_set(geo_tide, pts_src, pts_dest):
                    bounds_error=False, fill_value=0)
     rdr_tide = rgi_func(pts_dest)
     return rdr_tide
+
+
+def ionosphere_delay(sensing_time, wavelength,
+                     ionex_path, lon_arr, lat_arr, inc_arr):
+    '''
+    Calculate ionosphere delay for geolocation
+
+    Parameters
+    ----------
+    time_sensing: datetime.datetime
+        Sensing time of burst
+    wavelength: float
+        Wavelength of the signal
+    lon_arr: numpy.ndarray
+        array of longitude in radar grid
+    lat_arr: numpy.ndarray
+        array of latitude in radar grid
+    inc_arr: numpy.ndarray
+        array of incidence angle in radar grid
+
+    Returns
+    -------
+    slant_range_delay: np.ndarray
+        Ionospheric delay in slant range
+    '''
+
+    if not ionex_path:
+        raise RuntimeError('LUT correction was enabled, '
```

Comment:
Irrespective of the setting for the corrections, if `ionex_path` is not provided, we cannot perform the correction. The error message should reflect that and be independent of the correction settings. Also, let's check directly that the path exists

Reply:
I agree that the error message should be independent to the correction setting. Please take a look how the revised message looks like.

For file existence check, I think we might better to keep the current pattern because `os.path.exists(None)` will raise an error like below. Also this early-return pattern will make it easier to read.

> os.path.exists(None)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/jeong/miniconda3/lib/python3.9/genericpath.py", line 19, in exists
    os.stat(path)
TypeError: stat: path should be string, bytes, os.PathLike or integer, not NoneType

Reply:
Addressed as mentioned in the comment above.